chore(dcgw): Consolidate CIDR info (#5584)
Kong/developer.konghq.com: app/_includes/konnect/cidr-minimum-requirements.md

{:.danger}

> > **You cannot edit a Dedicated Cloud Gateway network CIDR:** To change the CIDR of a network, you must recreate the entire network. You cannot edit the CIDR of an existing network. 

Reviewer: I agree with this one.

> Before you create a Dedicated Cloud Gateway network, determine which CIDR range you want to use for your network.

Reviewer: ```suggestion
Before creating a Dedicated Cloud Gateway network, choose the CIDR range you want to use.
```

A CIDR block defines the range of IP addresses available for your Dedicated Cloud Gateway. 
If you're configuring private network connectivity, this CIDR block **must not** overlap with CIDR blocks assigned in your own cloud service provider networks to prevent conflicts.

> Keep in mind that your Dedicated Cloud Gateway network CIDR block must be large enough to cover the Kong infrastructure Kong will provision inside it, such as the data plane nodes, the DNS proxy, internal load balancers, and any other components Kong manages. 

Reviewer: ```suggestion
The CIDR block must also be large enough to accommodate all Kong-managed infrastructure provisioned inside the network such as the data plane nodes, the DNS proxy, internal load balancers, and other components.
```

> Keep the following CIDR requirements in mind when you're deciding your network CIDR range:

Reviewer: ```suggestion
Keep the following requirements in mind when choosing your network CIDR range:
```

> * **Prefix length:** The CIDR block must have a prefix length between `/16` and `/23`. `/23` blocks are only supported for up to 3 availability zones.

Reviewer: ```suggestion
* **Prefix length:** The CIDR block must have a prefix length between `/16` and `/23`. `/23` blocks support a maximum of 3 availability zones.
```
